build.gradle

```diff
@@ -15,27 +15,33 @@ apply plugin: 'org.springframework.boot'
 jar {
     archiveBaseName = 'bt-example'
     archiveVersion =  '0.1.0'
+    archiveClassifier = ""
```

Comment:
This and the manifest changes below were required to even get the default thin jar usable and named "the same" as prior to the dependency bumps.  Still requires dependency tango to work though, so it's easier to just use the springboot jar for compose.